src/middleware/inject_params.rs

```diff
@@ -133,13 +133,15 @@ mod tests {
     use serde_json::json;
     use tokio::sync::{mpsc, oneshot};
 
-    async fn create_client() -> (
-        Api,
-        ServerHandle,
-        mpsc::Receiver<(JsonValue, SubscriptionSink)>,
-        mpsc::Receiver<(JsonValue, SubscriptionSink)>,
-        mpsc::Receiver<(JsonValue, oneshot::Sender<JsonValue>)>,
-    ) {
+    struct ExecutionContext {
+        _server: ServerHandle,
```

Comment:
what is the underscore for?

Reply:
it's not directly used but shouldn't be dropped

Reply:
ok. I don't think underscore is required here. it is not something cannot be used

Reply:
clippy reports error if no underscore